templates/etc/fastd/fastd-blacklist.sh.erb

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+PEER_KEY=$1
+
+if /bin/grep -Fq $PEER_KEY /etc/fastd/<%= @mesh_code %>-mesh-vpn/fastd-blacklist.json; then
```

Comment:
why use a file in json format, which is hard to edit by hand, when we here grep within that file? then something like a one-key-per-line file would be totally sufficient